project/src/main/world/creature/emote-anims.gd

```diff
@@ -373,15 +364,6 @@ func _on_animation_finished(anim_name: String) -> void:
 		_post_unemote()
 
 
-"""
-Reset the eye frame when loading a new creature appearance.
-
-If we don't reset the eye frame, we have one strange transition frame.
-"""
-func _on_CreatureVisuals_before_creature_arrived() -> void:
-	_apply_default_eye_frames()
-
-
```

Comment:
Document this refactoring in the commit comment

Reply:
Expanded commit comment, `'reset frames' is now
handled exclusively by CreatureVisuals`...